src/js/common/schemaform/widgets/PhoneNumberWidget.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TextWidget from './TextWidget';
+
+/*
+ * Handles removing dashes from SSNs by keeping the user input in local state
+ * and saving the transformed version instead
+ */
+export default class PhoneNumberWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { val: props.value };
+  }
+  handleChange = (val) => {
+    const stripped = val.replace(/[\D]/g, '');
```

Comment:
I think it'd be better to switch this to exclude spaces, dashes, and parens only. If we strip other non-numeric characters, we're allowing through characters that are likely typed by accident and not telling the user.